start-here/js/script-main.js

```diff
@@ -18,10 +18,70 @@ function handleSearch() {
     input.classList.add("search-box--warning");
   } else {
     warningMessage.style.display = "none";
+    fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`)
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.title) {
+          document.getElementById(
+            "title-wrapper"
+          ).innerHTML = `<p>No definition found for <strong>${input.value}</strong>.</p>`;
+        } else {
+          const meanings = data[0].meanings;
+          const phonetics = data[0].phonetics;
+
+          document.querySelector(".title").innerText = input.value;
+
+          phonetics.forEach((phonetic) => {
+            if (phonetic.text) {
+              const h2 = document.querySelector("h2");
+              h2.innerText = phonetic.text;
+
+              const audio = document.querySelector("audio");
+              audio.src = phonetic.audio;
+            }
+          });
+
+          const resultWrappers = document.querySelectorAll(".result-wrapper");
+          resultWrappers.forEach((resultWrapper, index) => {
+            const meaning = meanings[index];
+            if (meaning) {
+              const gramType = resultWrapper.querySelector(
+                ".gramitical-type_wrapper h3"
+              );
+              gramType.innerText = meaning.partOfSpeech;
+
+              const meaningResultWrapper = resultWrapper.querySelector(
+                ".meaning-result_wrapper"
+              );
+              const listOfMeanings =
+                meaningResultWrapper.querySelector(".list-of-meanings");
+              listOfMeanings.innerHTML = "";
+
+              meaning.definitions.forEach((definition) => {
+                const li = document.createElement("li");
+                li.innerText = definition.definition;
+                listOfMeanings.appendChild(li);
+              });
+
+            }
+          });
+
```

Comment:
I miss the Synonyms and the example paragraph on the last meaning part of the Figma mockup. Should we add another issue for that?